apps/api/src/app/info/info.service.ts

```diff
@@ -199,16 +196,15 @@ export class InfoService {
 
   private async countGitHubStargazers(): Promise<number> {
     try {
-      const { stargazers_count } = await got(
+      const { stargazers_count } = await fetch(
```

Comment:
Better❓
```
const { stargazers_count }: { stargazers_count: number } = await fetch(
```

Reply:
I added the cast after the call to be consistent with other places. Also adding types on const is less convicing for some LSPs.